### DISCUSSION TITLE
Pull stored files via Rest calls

### DISCUSSION THREAD

**Discussion by gregmarnie** [pspete/psPAS#456] [Q&A]
Hello, new to all of this.  Curious if CyberArk CCP allows for files stored in a safe to be retrieved.

**Comment by pspete** [comment]
CCP will return details of a credential, as shown in the picture (if using the [CredentialRetriever](https://github.com/pspete/CredentialRetriever) module)
![image](https://user-images.githubusercontent.com/10600090/222518651-cdbf9643-0cea-48ba-b0bd-ee2c4e9d8003.png)
